tests/python/frontend/pytorch/test_forward.py

```diff
@@ -3147,18 +3147,28 @@ def __init__(self, dim=0):
         def forward(self, data, index, src):
             return torch.scatter(data, dim=self.dim, index=index, src=src)
 
+    class ScatterAdd(Module):
+        def __init__(self, dim=0):
+            super().__init__()
+            self.dim = dim
+
+        def forward(self, data, index, src):
+            return torch.scatter_add(data, dim=self.dim, index=index, src=src)
+
```

Comment:
As a general remark (rather than necessarily to act upon), I had hoped that supporting traced functions would help us make the test suite more lean... Somehow the module creation for each function looks unwieldy to me.

Reply:
I was not aware of traced function, indeed it made tests much cleaner.
We can enforce or encourage this style for new code.

Reply:
Glad it's not just obscure. Let's try encourage first. :slightly_smiling_face: